scipy/special/__init__.py

```diff
@@ -484,11 +484,7 @@
    :toctree: generated/
 
    lpmv                       -- Associated Legendre function of integer order and real degree.
```

Comment:
I didn't remove these other functions as they don't raise a deprecation warning

Reply:
@steppi what is the plan with these other functions? Should they be raising a warning?

Reply:
Sorry for missing the ping. The replacements haven't been written yet, so no warnings yet. It's something I've wanted to do but haven't had time for.